### PR TITLE
Adding version 3.0.3 of core-tools

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

File: cli-feed-v3.json [hunk 1]
```diff
@@ -2823,6 +2823,53 @@
           "sha2": "17e8ae6f0d372073581acf7b4d27622a2eaefd43cd60045b0ca2ca968ebd7dfa"
         }
       ]
+    },
+    "3.0.3": {
+      "Microsoft.NET.Sdk.Functions": "1.0.29",
```
Comment (brettsam) [inline cli-feed-v3.json]: 3.0.0-preview2?

File: cli-feed-v3.json [hunk 2]
```diff
@@ -2823,6 +2823,53 @@
...
+      "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/2.0.10344.zip",
+      "extensionBundle": {
+        "hostConfig": {
+          "extensionBundle": {
+            "id": "Microsoft.Azure.Functions.ExtensionBundle",
+            "version": "[1.*, 2.0.0)"
+          }
+        }
+      },
+      "FUNCTIONS_EXTENSION_VERSION": "~2",
```
Comment (brettsam) [inline cli-feed-v3.json]: ~3

File: cli-feed-v3.json [hunk 3]
```diff
@@ -2823,6 +2823,53 @@
...
+      "requiredRuntime": ".NET Core",
+      "minimumRuntimeVersion": "2.1",
```
Comment (brettsam) [inline cli-feed-v3.json]: 3.0?

File: cli-feed-v3.json (hunk 2 above)
Comment (soninaren) [reply cli-feed-v3.json]: fixed

File: cli-feed-v3.json (hunk 3 above)
Comment (soninaren) [reply cli-feed-v3.json]: fixed

File: cli-feed-v3.json (hunk 1 above)
Comment (soninaren) [reply cli-feed-v3.json]: fixed

**Pull request merged**